backend/app/main/views.py

```diff
@@ -68,5 +69,40 @@ def get_a_client(id):
             m = Client(name=name, notes=notes, attachments=attachments)
             client.append(m.serialize())
             return jsonify(client)
-    else: 
-        return "This client does not exist in the database."
\ No newline at end of file
+    else:
+        return "This client does not exist in the database."
+
+# Gets list of client notes based on clientid or email
+@main.route("/notes/<id>", methods=["GET"])
+def get_client_notes(id):
+    # Regex used to check is string input is an email address
+    regex = '^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$'
+    email_input = re.search(regex, id)
+
+    if (email_input):
+        # id is an email. Collect client notes by email
+        response = requests.get(
+            "https://api.airtable.com/v0/appw4RRMDig1g2PFI/Clients?filterByFormula=SEARCH('{}'".format(id) + ", {Client Email})",
+            headers={"Authorization": str(os.environ.get("API_KEY"))},
+        )
+    else:
+        # Not an email, assume it's an id
```

Comment:
Could we use regex to check if this is a valid id?

Reply:
Will do